scripts/prepare-package-for-release.js

```diff
@@ -49,6 +50,13 @@ const releaseVersion = argv.toVersion;
 const isLatest = argv.latest;
 const isDryRun = argv.dryRun;
 
+if (checkIfTagExists(releaseVersion)) {
+  echo(`Tag v${releaseVersion} already exists.`);
+  echo('You may want to rollback the last commit');
+  echo('git reset --hard HEAD~1');
+  exit(1);
+}
```

Comment:
can you also add this to `bump-oss-script`? After L46 would be perfect

Reply:
I'm not sure if at line 46 I can access `argv`. For what I can read, i should manually access `yargs`... I added it at line 78, right after the extraction of the version.

I also refactored it a little bit to avoid code duplication.